core/services/repo_manager.rb

```diff
@@ -83,6 +87,8 @@ def attribute_name(product)
 
   def find_repository(product_name, product, box)
     @ui.info('Looking for repo')
+    return { 'version' => product['version'] } if PRODUCT_ATTRIBUTES[product_name][:repository].nil?
```

Comment:
Please provide warning to the stdout that will state that we can not determine whether the version is correct or not.